### DISCUSSION TITLE
MIMO_OFDM_Transmissions_over_CDL tutorial question

### DISCUSSION THREAD

**Discussion by thibaut29** [NVlabs/sionna#141] [Q&A] [answered]
Hello everyone,

I'm doing the MIMO_OFDM_Transmissions_over_CDL tutorial and I don't understand everything.
Indeed, because we have more than 1 stream (number of UT antenna) => the function **ebnodb2no()** divides the energy per symbol by the number of transmitted streams : Logic if the total of transmitted power is divided bythe number of transmitted stream to ensure fair comparaison with SISO system. But later in the code, I don't see any power normalization.
Is it normal? 
If yes, may anyone explain why? 

Thank you in advance,

note : Thank you for all the work done in SIONNA and the quality of the API. It's amazing :)

**Comment by jhoydis** [comment]
Hi,

The function ``ebnodb2no`` returns the noise power that the channel should apply to achieve a target Eb/No. By default, all signals in Sionna are normalised to unit power. Therefore, you do not see any additional power normalisation. Hope that helps your understanding.

**Reply by thibaut29** [reply]
Thank you for your answer. 

I'm sorry but I still don't understand.

Suppose power of each stream is 1.
More Tx antenna there is, more the noise power should be high to offset the fact that total transmitted power at each received antenna is higher. But what I see when increasing nuber of Tx antenna is that No returned by **ebnodb2no** decreases. 
It's like the function suppose that we divide power of each stream by the number of streams.

**Reply by jhoydis** [reply, marked as answer]
Hi, I have looked into it and now understand your concern. The power per transmit stream in this notebook is equal to one. The function ``ebnodb2no`` assumes, however, that each stream only gets an equal fraction of this power. I need to carefully think about why we introduced this scaling, but it might simply be wrong. I'll come back to you once this is sorted out. A simple fix would be to divide the precoded signal by the square-root of the number of streams per transmitter.

**Reply by thibaut29** [reply]
Yes It's what I'm doing.

Thank you
